.ci/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {label "clx-host-21514||clx-host-21518"}
```

Comment:
maybe best to use label name based on capabilities rather hostnames
for portability.

Reply:
Changed to the Jenkins label "ml-test-node-gpu"